test/functional/wallet_rescan_unconfirmed.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that descriptor wallets rescan mempool transactions properly when importing."""
+
+from test_framework.address import (
+    address_to_scriptpubkey,
+    ADDRESS_BCRT1_UNSPENDABLE,
+)
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from test_framework.wallet import MiniWallet
+from test_framework.wallet_util import test_address
+
+
+class WalletRescanUnconfirmed(BitcoinTestFramework):
+    def add_options(self, parser):
+        self.add_wallet_options(parser, legacy=False)
+
+    def set_test_params(self):
+        self.num_nodes = 3
+        # Immediate tx relay
+        self.extra_args = [['-whitelist=noban@127.0.0.1']] * self.num_nodes
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+        self.skip_if_no_sqlite()
+
+    def run_test(self):
+        # Node with which we will use to send funds to the wallet and mine blocks
+        tester_node = self.nodes[0]
+        # Node on which the wallet is originally created
+        orig_wallet_node = self.nodes[1]
+        # Node which will import the wallet and rescan blocks / mempool for transactions
+        import_wallet_node = self.nodes[2]
+
+
+        self.log.info("Create wallets and mine initial chain")
+        tester_wallet = MiniWallet(tester_node)
+        self.generate(tester_wallet, 110)
```

Comment:
Is there a reason it's 110 and not `COINBASE_MATURITY`?

Reply:
changed